parser/SqlParser.ypp

```diff
@@ -759,6 +780,51 @@ opt_table_constraint_def_commalist:
     /* $$ = $1; */
   };
 
+opt_column_list:
+  /* empty */
+  | '(' name_commalist ')' {
+    delete $2;
+    NotSupported(&@1, yyscanner, "list of column names in CREATE INDEX statement");
+    YYERROR;
+  };
+
+index_type:
+  TOKEN_BLOOM_FILTER_INDEX {
+    NotSupported(&@1, yyscanner, "Bloom Filter Index");
+    YYERROR;
+  };
+
+opt_index_properties:
+  /* empty */
+  | '(' index_properties_list ')' {
+    NotSupported(&@1, yyscanner, "list of index properties in CREATE INDEX statement");
+    YYERROR;
+  };
+
+index_properties_list:
+  index_property {
+    NotSupported(&@1, yyscanner, "index property in CREATE INDEX statement");
+    YYERROR;
+  }
+  | index_properties_list ',' index_property {
+    NotSupported(&@1, yyscanner, "list of index properties in CREATE INDEX statement");
+    YYERROR;
+  };
+
+index_property:
+  any_name literal_value {
```

Comment:
Yeah, we will soon replace this with a more generic key-value configuration that is sort of reusable across multiple statements.